src/ui/Main.tsx

```diff
@@ -121,11 +176,66 @@ function Main() {
 
   const importFn = useCallback((json: string) => {
     const result = requestStore.import(json);
+
     setSpecEndpoints();
     setAllHosts(new Set(requestStore.hosts()));
     return result;
   }, []);
 
+
+  const describeSelectedEndpoints = async (selectedEndpoints: Set<string>) => {
+
+    const descriptions = requestStore.getEndpointDescriptions();
+    const requestBodySchemaParams = requestStore.requestBodySchemaParamDescriptions;
+    const responseBodySchemaParams = requestStore.responseBodySchemaParamDescriptions;
+    const queryParams = requestStore.queryParamDescriptions;
+
+    setDescriptionsLoading(DescriptionStatus.ACTIVE);
+
+    for (const id of selectedEndpoints) {
```

Comment:
Still doing these in series so it's really slow when it doesn't need to be :smile: